### DISCUSSION TITLE
Buzz's license of use

### DISCUSSION THREAD

**Discussion by msgc24** [chidiwilliams/buzz#1011] [Q&A]
Hi chidiwilliams,

I find Buzz really useful! I just was wondering if its license allows its use for scientific research and institutional purposes and is not limited to personal uses. I have reviewed the MIT License, but I would like to confirm whether its terms permit the described uses beyond personal use. Thanks!

**Comment by raivisdejus** [comment]
@msgc24 Hi! Glad you find Buzz useful :)

I am contributor in this project and my understanding of opens source licenses is such that you can use Buzz for whatever purpose you need as private person, as research institution or even as company for commercial purposes. 

If you wish you can even change the source code and adjust is as you need. It is all permitted.

My personal request (not part of the license) is that if you add some useful new feature, you share it back with us. Or if you find some bug, please report it, so we can fix it.
